guides/v2.3/graphql/reference/checkout-aggreements.md

```diff
@@ -0,0 +1,71 @@
+---
+group: graphql
+title: Directory endpoint
+contributor_name: Something Digital
+contributor_link: https://www.somethingdigital.com/
+---
+
+To retrieve enabled checkout agreements. The query will always return an empty array when
+`checkout/options/enable_agreements` is disabled.
+
```

Comment:
Create a head2 labeled Output attributes with the following sentence:

The `CheckoutAgreements` object contains an array of `CheckoutAgreement` objects.

(or something else, if you can think of anything better to say).

Move the Output attributes section to the end.

Reply:
Can you take a look and make sure this is the intended result? https://github.com/magento/devdocs/pull/4943/commits/dbd8ee2ced219112516709d2564a31238e4b2051#diff-b903ec86dd935702a7085d632c676d8dR69